content/_data/changelogs/weekly.yml

```
@@ -17914,7 +17914,96 @@
   # pull: 7227 (PR title: Update dependency css-minimizer-webpack-plugin to v4.2.1)
   # pull: 7228 (PR title: Update dependency less-loader to v11.1.0)
   # pull: 7234 (PR title: Update dependency eslint to v8.25.0)
-  
+
+  - version: '2.374'
+    date: 2022-10-16
+    changes:
+
+      - type: rfe
+        category: rfe
+        pull: 7091
+        authors:
+          - omehegan
+        pr_title: Clarify safe restart won't wait for Pipeline jobs
+        message: |-
+          Clarify safe restart won't wait for Pipeline jobs.
+      - type: rfe
+        category: rfe
+        pull: 6951
+        authors:
+          - mawinter69
+          - basil
+        pr_title: Allow Formchecker to check more than 1 thing at a time
+        message: |-
+          Allow Formchecker to check more than one thing at a time.
+      - type: rfe
+        category: rfe
+        pull: 7205
+        issue: 69714
+        authors:
+          - kaltepeter
+        pr_title: "[JENKINS-69714] Use new jenkins-table for slow trigger admin monitor"
+        message: |-
+          Replace the old Jenkins table layout in the slow trigger admin monitor with the new Jenkins table layout.
```

"admin" is an abbreviation, not a proper English word:

```suggestion
          Replace the old Jenkins table layout in the slow trigger administrative monitor with the new Jenkins table layout.
```